### PR TITLE
[Grid Config] Admins should be able to change + delete shared grid configurations

### PR DESCRIPTION
**Pull request by BlackbitNeueMedien**
Currently only the owner of a shared grid configuration (aka the person who created the grid config) can change and delete it. With this PR also admin users are allowed to change and delete shared grid configurations.

Additionally this PR adds 2 readonly fields in the grid config `Save & Share` panel for the owner and the modification timestamp. This way users can find out the owner of a shared grid config if they want to suggest changes.

### CODE REVIEW

**Review by dvesh3 — CHANGES_REQUESTED** [review]
still not able to delete shared config by admin. see below:
image

File: bundles/CoreBundle/Resources/translations/en.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
   "mimetype": "MIME-Type",
   "creationdate": "Creation Date",
   "usermodification": "User Modification",
-  "userowner": "User Owner",
+  "userowner": "Owner",
```
[the remark this answers is not in the document]
Comment (BlackbitNeueMedien) [reply bundles/CoreBundle/Resources/translations/en.json:43]: Is `User Owner` clean english? In german this gets translated as `Eigentümer des Benutzers` (= owner of user) - and this is not a proper label for this field.

**Comment by dvesh3** [comment]
@BlackbitNeueMedien please see https://github.com/pimcore/pimcore/pull/8549#pullrequestreview-619454953

**Comment by BlackbitNeueMedien** [comment]
@dvesh3 could you tell me how to reproduce your state?
I just tried it again by doing the following:
1. Ensure you are logged-in with user `admin` who is an administrator
2. Go to a data object folder, click `Grid Options`, go to `Save & Share` panel, enter `name` "test", check `Share globally` and click `Copy & Share` button
3. Create user `test` who is also an administrator
4. log in with user `test` go to folder from 2., select `test` in `Grid Options` button's dropdown
5. Expand `Grid Options` button's dropdown again and in my case `Delete` is possible

**Comment by dvesh3** [comment]
@BlackbitNeueMedien I used `shared user` option to share with administrator user, instead of `Share globally`.

**Comment by brusch** [comment]
@BlackbitNeueMedien  any updates on this? Thanks 😊

**Comment by dvesh3** [comment]
🏓 @BlackbitNeueMedien

**Comment by BlackbitNeueMedien** [comment]
@dvesh3 Should work now also for configurations which are shared with users or roles (and not globally).

**Review by dvesh3 — APPROVED** [review]

**Comment by dvesh3** [comment]
@BlackbitNeueMedien thanks a lot!

**Pull request merged**